nss-cohort-37/Workforce-Blue-Barracudas
# Nm computers view

# Description
Computer controller list, delete, create. Along with assigning employee to the computer. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Click on Computers and you will see a list of computers. Add a new computer and either choose an employee or do not. If you do not select and employee the computer will have a delete button. If you click details you can see more about the computer. 
# Checklist:
